### ISSUE TITLE
Row headers not specified

### ISSUE THREAD

**Issue by anoQlik** [qlik-oss/sn-table#217]
The first cell in each row should be marked up as a <th> and be given a scope of "row."

**Comment by haxxmaxx** [comment]
I think I mentioned this elsewhere, not sure we should have a row header, since the first column isn't necessarily the header, even though it usually is used that way.

**Comment by haxxmaxx** [comment]
Adding it to the new board. Probably can be closed, but it needs to be adressed.

**Comment by haxxmaxx** [comment]
Even though there might be corner-cases where this becomes weird, let's use the first column as row header.

**Comment by haxxmaxx** [comment]
This wasn't fixed properly fixed, should use <th> instead of <td>.
